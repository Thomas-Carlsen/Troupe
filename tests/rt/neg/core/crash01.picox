let fun crash () = let val x = [] 
                   in case x of     
                            (10::[]) => () 
                   end 
 
    fun loop i = let val _ = print i 
                 in loop ( i + 1 ) 
                 end 
    val _ = spawn crash   
in ()
end 
