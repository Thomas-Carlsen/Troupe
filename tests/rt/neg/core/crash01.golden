2019-03-27T19:47:41.810Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
>>> Main thread finished with value: ()@{}%{}
[31mRuntime error in thread 202c1337-4cee-4d5d-81c6-e2352465cb06@{}%{}[39m
[31m>> pattern match failure in case expression at :2:23[39m
